syft/__init__.py

```diff
@@ -25,15 +25,23 @@
 # Tensorflow / Keras dependencies
 # Import Hooks
 
+__all__ = []
 if dependency_check.tfe_available:
     from syft.frameworks.keras import KerasHook
     from syft.workers.tfe import TFECluster
     from syft.workers.tfe import TFEWorker
 
-    __all__ = ["KerasHook", "TFECluster", "TFEWorker"]
+    __all__.extend(["KerasHook", "TFECluster", "TFEWorker"])
 else:
     logger.info("TF Encrypted Keras not available.")
-    __all__ = []
+
+if dependency_check.crypten_available:
+    from syft.frameworks.torch.tensors.crypten.syft_crypten import SyftCrypTensor
+
+    __all__.extend(["SyftCrypTensor"])
```

Comment:
I'm not sure about this, but if SyftCrypTensor is a tensor type that its only purpose is for building plans with crypten functionalities, then should we make it available in the API?

Reply:
Removed it